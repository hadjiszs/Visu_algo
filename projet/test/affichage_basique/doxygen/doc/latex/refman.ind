\begin{theindex}

  \item {cellule}, \hyperpage{7}
  \item {charger}
    \subitem {Graphe}, \hyperpage{11}
    \subitem {Shader}, \hyperpage{22}
    \subitem {Structure}, \hyperpage{25}

  \indexspace

  \item {del}
    \subitem {Shader}, \hyperpage{22}

  \indexspace

  \item {File}, \hyperpage{8}

  \indexspace

  \item {Graphe}, \hyperpage{8}
    \subitem {charger}, \hyperpage{11}
    \subitem {Graphe}, \hyperpage{10}

  \indexspace

  \item {Liste}, \hyperpage{12}

  \indexspace

  \item {Shader}, \hyperpage{20}
    \subitem {charger}, \hyperpage{22}
    \subitem {del}, \hyperpage{22}
    \subitem {Shader}, \hyperpage{20}
  \item {sommet}, \hyperpage{23}
  \item {Structure}, \hyperpage{23}
    \subitem {charger}, \hyperpage{25}
    \subitem {Structure}, \hyperpage{24}

\end{theindex}
