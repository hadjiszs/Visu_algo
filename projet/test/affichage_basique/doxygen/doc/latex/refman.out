\BOOKMARK [0][-]{chapter.1}{\376\377\0001\000\040\000I\000n\000d\000e\000x\000\040\000h\000i\000\351\000r\000a\000r\000c\000h\000i\000q\000u\000e}{}% 1
\BOOKMARK [1][-]{section.1.1}{\376\377\0001\000.\0001\000\040\000H\000i\000\351\000r\000a\000r\000c\000h\000i\000e\000\040\000d\000e\000s\000\040\000c\000l\000a\000s\000s\000e\000s}{chapter.1}% 2
\BOOKMARK [0][-]{chapter.2}{\376\377\0002\000\040\000I\000n\000d\000e\000x\000\040\000d\000e\000s\000\040\000c\000l\000a\000s\000s\000e\000s}{}% 3
\BOOKMARK [1][-]{section.2.1}{\376\377\0002\000.\0001\000\040\000L\000i\000s\000t\000e\000\040\000d\000e\000s\000\040\000c\000l\000a\000s\000s\000e\000s}{chapter.2}% 4
\BOOKMARK [0][-]{chapter.3}{\376\377\0003\000\040\000I\000n\000d\000e\000x\000\040\000d\000e\000s\000\040\000f\000i\000c\000h\000i\000e\000r\000s}{}% 5
\BOOKMARK [1][-]{section.3.1}{\376\377\0003\000.\0001\000\040\000L\000i\000s\000t\000e\000\040\000d\000e\000s\000\040\000f\000i\000c\000h\000i\000e\000r\000s}{chapter.3}% 6
\BOOKMARK [0][-]{chapter.4}{\376\377\0004\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000l\000a\000s\000s\000e\000s}{}% 7
\BOOKMARK [1][-]{section.4.1}{\376\377\0004\000.\0001\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000c\000e\000l\000l\000u\000l\000e}{chapter.4}% 8
\BOOKMARK [2][-]{subsection.4.1.1}{\376\377\0004\000.\0001\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.1}% 9
\BOOKMARK [1][-]{section.4.2}{\376\377\0004\000.\0002\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000F\000i\000l\000e}{chapter.4}% 10
\BOOKMARK [2][-]{subsection.4.2.1}{\376\377\0004\000.\0002\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.2}% 11
\BOOKMARK [1][-]{section.4.3}{\376\377\0004\000.\0003\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000G\000r\000a\000p\000h\000e}{chapter.4}% 12
\BOOKMARK [2][-]{subsection.4.3.1}{\376\377\0004\000.\0003\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.3}% 13
\BOOKMARK [2][-]{subsection.4.3.2}{\376\377\0004\000.\0003\000.\0002\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000e\000u\000r\000s\000\040\000e\000t\000\040\000d\000e\000s\000t\000r\000u\000c\000t\000e\000u\000r}{section.4.3}% 14
\BOOKMARK [3][-]{subsubsection.4.3.2.1}{\376\377\0004\000.\0003\000.\0002\000.\0001\000\040\000G\000r\000a\000p\000h\000e}{subsection.4.3.2}% 15
\BOOKMARK [2][-]{subsection.4.3.3}{\376\377\0004\000.\0003\000.\0003\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000o\000n\000c\000t\000i\000o\000n\000s\000\040\000m\000e\000m\000b\000r\000e\000s}{section.4.3}% 16
\BOOKMARK [3][-]{subsubsection.4.3.3.1}{\376\377\0004\000.\0003\000.\0003\000.\0001\000\040\000a\000f\000f\000i\000c\000h\000e\000r\000\137\000c\000h\000e\000m\000i\000n}{subsection.4.3.3}% 17
\BOOKMARK [3][-]{subsubsection.4.3.3.2}{\376\377\0004\000.\0003\000.\0003\000.\0002\000\040\000a\000f\000f\000i\000c\000h\000e\000r\000\137\000l\000i\000s\000t\000e\000s\000\137\000a\000d\000j\000a\000c\000e\000n\000c\000e\000s}{subsection.4.3.3}% 18
\BOOKMARK [3][-]{subsubsection.4.3.3.3}{\376\377\0004\000.\0003\000.\0003\000.\0003\000\040\000a\000f\000f\000i\000c\000h\000e\000r\000\137\000m\000a\000t\000r\000i\000c\000e\000\137\000a\000d\000j\000a\000c\000e\000n\000c\000e\000s}{subsection.4.3.3}% 19
\BOOKMARK [3][-]{subsubsection.4.3.3.4}{\376\377\0004\000.\0003\000.\0003\000.\0004\000\040\000a\000f\000f\000i\000c\000h\000e\000r\000\137\000p\000a\000r\000c\000o\000u\000r\000s\000\137\000p\000r\000o\000f\000o\000n\000d\000e\000u\000r}{subsection.4.3.3}% 20
\BOOKMARK [3][-]{subsubsection.4.3.3.5}{\376\377\0004\000.\0003\000.\0003\000.\0005\000\040\000c\000h\000a\000r\000g\000e\000r}{subsection.4.3.3}% 21
\BOOKMARK [3][-]{subsubsection.4.3.3.6}{\376\377\0004\000.\0003\000.\0003\000.\0006\000\040\000D\000F\000S\000\137\000v\000i\000s\000i\000t\000e\000r\000\137\000n\000o\000e\000u\000d}{subsection.4.3.3}% 22
\BOOKMARK [3][-]{subsubsection.4.3.3.7}{\376\377\0004\000.\0003\000.\0003\000.\0007\000\040\000p\000a\000r\000c\000o\000u\000r\000s\000\137\000l\000a\000r\000g\000e\000u\000r}{subsection.4.3.3}% 23
\BOOKMARK [3][-]{subsubsection.4.3.3.8}{\376\377\0004\000.\0003\000.\0003\000.\0008\000\040\000p\000a\000r\000c\000o\000u\000r\000s\000\137\000p\000r\000o\000f\000o\000n\000d\000e\000u\000r}{subsection.4.3.3}% 24
\BOOKMARK [1][-]{section.4.4}{\376\377\0004\000.\0004\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000L\000i\000s\000t\000e}{chapter.4}% 25
\BOOKMARK [2][-]{subsection.4.4.1}{\376\377\0004\000.\0004\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.4}% 26
\BOOKMARK [1][-]{section.4.5}{\376\377\0004\000.\0005\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000l\000i\000s\000t\000e\000\137\000a\000d\000j\000a\000c\000e\000n\000c\000e\000\137\000t}{chapter.4}% 27
\BOOKMARK [2][-]{subsection.4.5.1}{\376\377\0004\000.\0005\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.5}% 28
\BOOKMARK [1][-]{section.4.6}{\376\377\0004\000.\0006\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000M\000a\000i\000n\000W\000i\000n\000d\000o\000w}{chapter.4}% 29
\BOOKMARK [2][-]{subsection.4.6.1}{\376\377\0004\000.\0006\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.6}% 30
\BOOKMARK [2][-]{subsection.4.6.2}{\376\377\0004\000.\0006\000.\0002\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000e\000u\000r\000s\000\040\000e\000t\000\040\000d\000e\000s\000t\000r\000u\000c\000t\000e\000u\000r}{section.4.6}% 31
\BOOKMARK [3][-]{subsubsection.4.6.2.1}{\376\377\0004\000.\0006\000.\0002\000.\0001\000\040\000M\000a\000i\000n\000W\000i\000n\000d\000o\000w}{subsection.4.6.2}% 32
\BOOKMARK [2][-]{subsection.4.6.3}{\376\377\0004\000.\0006\000.\0003\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000o\000n\000c\000t\000i\000o\000n\000s\000\040\000m\000e\000m\000b\000r\000e\000s}{section.4.6}% 33
\BOOKMARK [3][-]{subsubsection.4.6.3.1}{\376\377\0004\000.\0006\000.\0003\000.\0001\000\040\000c\000h\000a\000r\000g\000e\000r\000\137\000m\000e\000n\000u}{subsection.4.6.3}% 34
\BOOKMARK [3][-]{subsubsection.4.6.3.2}{\376\377\0004\000.\0006\000.\0003\000.\0002\000\040\000c\000h\000a\000r\000g\000e\000r\000\137\000o\000p\000e\000n\000g\000l}{subsection.4.6.3}% 35
\BOOKMARK [3][-]{subsubsection.4.6.3.3}{\376\377\0004\000.\0006\000.\0003\000.\0003\000\040\000s\000e\000l\000e\000c\000t\000i\000o\000n\000\137\000g\000r\000a\000p\000h\000e}{subsection.4.6.3}% 36
\BOOKMARK [1][-]{section.4.7}{\376\377\0004\000.\0007\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000m\000a\000t\000r\000i\000c\000e\000\137\000a\000d\000j\000a\000c\000e\000n\000c\000e\000\137\000t}{chapter.4}% 37
\BOOKMARK [2][-]{subsection.4.7.1}{\376\377\0004\000.\0007\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.7}% 38
\BOOKMARK [1][-]{section.4.8}{\376\377\0004\000.\0008\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000m\000a\000t\000r\000i\000c\000e\000\137\000l\000a\000p\000l\000a\000c\000e\000\137\000t}{chapter.4}% 39
\BOOKMARK [2][-]{subsection.4.8.1}{\376\377\0004\000.\0008\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.8}% 40
\BOOKMARK [1][-]{section.4.9}{\376\377\0004\000.\0009\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000p\000a\000r\000c\000o\000u\000r\000s\000\137\000t}{chapter.4}% 41
\BOOKMARK [2][-]{subsection.4.9.1}{\376\377\0004\000.\0009\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.9}% 42
\BOOKMARK [1][-]{section.4.10}{\376\377\0004\000.\0001\0000\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000S\000c\000e\000n\000e\000G\000L}{chapter.4}% 43
\BOOKMARK [2][-]{subsection.4.10.1}{\376\377\0004\000.\0001\0000\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.10}% 44
\BOOKMARK [2][-]{subsection.4.10.2}{\376\377\0004\000.\0001\0000\000.\0002\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000e\000u\000r\000s\000\040\000e\000t\000\040\000d\000e\000s\000t\000r\000u\000c\000t\000e\000u\000r}{section.4.10}% 45
\BOOKMARK [3][-]{subsubsection.4.10.2.1}{\376\377\0004\000.\0001\0000\000.\0002\000.\0001\000\040\000S\000c\000e\000n\000e\000G\000L}{subsection.4.10.2}% 46
\BOOKMARK [3][-]{subsubsection.4.10.2.2}{\376\377\0004\000.\0001\0000\000.\0002\000.\0002\000\040\000S\000c\000e\000n\000e\000G\000L}{subsection.4.10.2}% 47
\BOOKMARK [2][-]{subsection.4.10.3}{\376\377\0004\000.\0001\0000\000.\0003\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000o\000n\000c\000t\000i\000o\000n\000s\000\040\000m\000e\000m\000b\000r\000e\000s}{section.4.10}% 48
\BOOKMARK [3][-]{subsubsection.4.10.3.1}{\376\377\0004\000.\0001\0000\000.\0003\000.\0001\000\040\000c\000h\000a\000r\000g\000e\000r\000\137\000c\000o\000n\000t\000e\000n\000u\000\137\000g\000r\000a\000p\000h\000i\000q\000u\000e}{subsection.4.10.3}% 49
\BOOKMARK [3][-]{subsubsection.4.10.3.2}{\376\377\0004\000.\0001\0000\000.\0003\000.\0002\000\040\000c\000l\000e\000a\000n\000u\000p\000G\000L}{subsection.4.10.3}% 50
\BOOKMARK [3][-]{subsubsection.4.10.3.3}{\376\377\0004\000.\0001\0000\000.\0003\000.\0003\000\040\000g\000e\000t\000\137\000s\000t\000r\000u\000c\000t\000u\000r\000e}{subsection.4.10.3}% 51
\BOOKMARK [3][-]{subsubsection.4.10.3.4}{\376\377\0004\000.\0001\0000\000.\0003\000.\0004\000\040\000i\000n\000i\000t\000i\000a\000l\000i\000z\000e\000G\000L}{subsection.4.10.3}% 52
\BOOKMARK [3][-]{subsubsection.4.10.3.5}{\376\377\0004\000.\0001\0000\000.\0003\000.\0005\000\040\000p\000a\000i\000n\000t\000G\000L}{subsection.4.10.3}% 53
\BOOKMARK [3][-]{subsubsection.4.10.3.6}{\376\377\0004\000.\0001\0000\000.\0003\000.\0006\000\040\000r\000e\000s\000i\000z\000e\000G\000L}{subsection.4.10.3}% 54
\BOOKMARK [3][-]{subsubsection.4.10.3.7}{\376\377\0004\000.\0001\0000\000.\0003\000.\0007\000\040\000Z\000o\000o\000m\000n\000e\000g}{subsection.4.10.3}% 55
\BOOKMARK [3][-]{subsubsection.4.10.3.8}{\376\377\0004\000.\0001\0000\000.\0003\000.\0008\000\040\000Z\000o\000o\000m\000p\000o\000s}{subsection.4.10.3}% 56
\BOOKMARK [1][-]{section.4.11}{\376\377\0004\000.\0001\0001\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000S\000c\000e\000n\000e\000V\000e\000r\000t\000e\000x}{chapter.4}% 57
\BOOKMARK [2][-]{subsection.4.11.1}{\376\377\0004\000.\0001\0001\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.11}% 58
\BOOKMARK [1][-]{section.4.12}{\376\377\0004\000.\0001\0002\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000S\000h\000a\000d\000e\000r}{chapter.4}% 59
\BOOKMARK [2][-]{subsection.4.12.1}{\376\377\0004\000.\0001\0002\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.12}% 60
\BOOKMARK [2][-]{subsection.4.12.2}{\376\377\0004\000.\0001\0002\000.\0002\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000e\000u\000r\000s\000\040\000e\000t\000\040\000d\000e\000s\000t\000r\000u\000c\000t\000e\000u\000r}{section.4.12}% 61
\BOOKMARK [3][-]{subsubsection.4.12.2.1}{\376\377\0004\000.\0001\0002\000.\0002\000.\0001\000\040\000S\000h\000a\000d\000e\000r}{subsection.4.12.2}% 62
\BOOKMARK [3][-]{subsubsection.4.12.2.2}{\376\377\0004\000.\0001\0002\000.\0002\000.\0002\000\040\000S\000h\000a\000d\000e\000r}{subsection.4.12.2}% 63
\BOOKMARK [2][-]{subsection.4.12.3}{\376\377\0004\000.\0001\0002\000.\0003\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000o\000n\000c\000t\000i\000o\000n\000s\000\040\000m\000e\000m\000b\000r\000e\000s}{section.4.12}% 64
\BOOKMARK [3][-]{subsubsection.4.12.3.1}{\376\377\0004\000.\0001\0002\000.\0003\000.\0001\000\040\000c\000h\000a\000r\000g\000e\000r}{subsection.4.12.3}% 65
\BOOKMARK [3][-]{subsubsection.4.12.3.2}{\376\377\0004\000.\0001\0002\000.\0003\000.\0002\000\040\000d\000e\000l}{subsection.4.12.3}% 66
\BOOKMARK [3][-]{subsubsection.4.12.3.3}{\376\377\0004\000.\0001\0002\000.\0003\000.\0003\000\040\000v\000e\000r\000i\000f\000\137\000c\000o\000m\000p\000i\000l\000\137\000s\000h\000a\000d\000e\000r}{subsection.4.12.3}% 67
\BOOKMARK [3][-]{subsubsection.4.12.3.4}{\376\377\0004\000.\0001\0002\000.\0003\000.\0004\000\040\000v\000e\000r\000i\000f\000\137\000l\000i\000n\000k\000\137\000s\000h\000a\000d\000e\000r}{subsection.4.12.3}% 68
\BOOKMARK [1][-]{section.4.13}{\376\377\0004\000.\0001\0003\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000\040\000s\000o\000m\000m\000e\000t}{chapter.4}% 69
\BOOKMARK [2][-]{subsection.4.13.1}{\376\377\0004\000.\0001\0003\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.13}% 70
\BOOKMARK [1][-]{section.4.14}{\376\377\0004\000.\0001\0004\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000e\000\040\000l\000a\000\040\000c\000l\000a\000s\000s\000e\000\040\000S\000t\000r\000u\000c\000t\000u\000r\000e}{chapter.4}% 71
\BOOKMARK [2][-]{subsection.4.14.1}{\376\377\0004\000.\0001\0004\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.4.14}% 72
\BOOKMARK [2][-]{subsection.4.14.2}{\376\377\0004\000.\0001\0004\000.\0002\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000c\000o\000n\000s\000t\000r\000u\000c\000t\000e\000u\000r\000s\000\040\000e\000t\000\040\000d\000e\000s\000t\000r\000u\000c\000t\000e\000u\000r}{section.4.14}% 73
\BOOKMARK [3][-]{subsubsection.4.14.2.1}{\376\377\0004\000.\0001\0004\000.\0002\000.\0001\000\040\000S\000t\000r\000u\000c\000t\000u\000r\000e}{subsection.4.14.2}% 74
\BOOKMARK [2][-]{subsection.4.14.3}{\376\377\0004\000.\0001\0004\000.\0003\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000o\000n\000c\000t\000i\000o\000n\000s\000\040\000m\000e\000m\000b\000r\000e\000s}{section.4.14}% 75
\BOOKMARK [3][-]{subsubsection.4.14.3.1}{\376\377\0004\000.\0001\0004\000.\0003\000.\0001\000\040\000c\000h\000a\000r\000g\000e\000r}{subsection.4.14.3}% 76
\BOOKMARK [3][-]{subsubsection.4.14.3.2}{\376\377\0004\000.\0001\0004\000.\0003\000.\0002\000\040\000c\000o\000m\000p\000u\000t\000e\000\137\000c\000o\000o\000r\000d\000o\000n\000n\000e\000s}{subsection.4.14.3}% 77
\BOOKMARK [3][-]{subsubsection.4.14.3.3}{\376\377\0004\000.\0001\0004\000.\0003\000.\0003\000\040\000e\000s\000t\000\137\000i\000n\000i\000t}{subsection.4.14.3}% 78
\BOOKMARK [0][-]{chapter.5}{\376\377\0005\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n\000\040\000d\000e\000s\000\040\000f\000i\000c\000h\000i\000e\000r\000s}{}% 79
\BOOKMARK [1][-]{section.5.1}{\376\377\0005\000.\0001\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000g\000r\000a\000p\000h\000e\000.\000c\000p\000p}{chapter.5}% 80
\BOOKMARK [2][-]{subsection.5.1.1}{\376\377\0005\000.\0001\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.1}% 81
\BOOKMARK [1][-]{section.5.2}{\376\377\0005\000.\0002\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000g\000r\000a\000p\000h\000e\000.\000h\000p\000p}{chapter.5}% 82
\BOOKMARK [2][-]{subsection.5.2.1}{\376\377\0005\000.\0002\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.2}% 83
\BOOKMARK [1][-]{section.5.3}{\376\377\0005\000.\0003\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000m\000a\000i\000n\000.\000c\000p\000p}{chapter.5}% 84
\BOOKMARK [2][-]{subsection.5.3.1}{\376\377\0005\000.\0003\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.3}% 85
\BOOKMARK [1][-]{section.5.4}{\376\377\0005\000.\0004\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000m\000a\000i\000n\000w\000i\000n\000d\000o\000w\000.\000c\000p\000p}{chapter.5}% 86
\BOOKMARK [2][-]{subsection.5.4.1}{\376\377\0005\000.\0004\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.4}% 87
\BOOKMARK [1][-]{section.5.5}{\376\377\0005\000.\0005\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000m\000a\000i\000n\000w\000i\000n\000d\000o\000w\000.\000h\000p\000p}{chapter.5}% 88
\BOOKMARK [2][-]{subsection.5.5.1}{\376\377\0005\000.\0005\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.5}% 89
\BOOKMARK [1][-]{section.5.6}{\376\377\0005\000.\0006\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000c\000e\000n\000e\000.\000c\000p\000p}{chapter.5}% 90
\BOOKMARK [2][-]{subsection.5.6.1}{\376\377\0005\000.\0006\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.6}% 91
\BOOKMARK [1][-]{section.5.7}{\376\377\0005\000.\0007\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000c\000e\000n\000e\000.\000h\000p\000p}{chapter.5}% 92
\BOOKMARK [2][-]{subsection.5.7.1}{\376\377\0005\000.\0007\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.7}% 93
\BOOKMARK [1][-]{section.5.8}{\376\377\0005\000.\0008\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000h\000a\000d\000e\000r\000.\000c\000p\000p}{chapter.5}% 94
\BOOKMARK [2][-]{subsection.5.8.1}{\376\377\0005\000.\0008\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.8}% 95
\BOOKMARK [1][-]{section.5.9}{\376\377\0005\000.\0009\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000h\000a\000d\000e\000r\000.\000h\000p\000p}{chapter.5}% 96
\BOOKMARK [2][-]{subsection.5.9.1}{\376\377\0005\000.\0009\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.9}% 97
\BOOKMARK [1][-]{section.5.10}{\376\377\0005\000.\0001\0000\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000.\000c\000p\000p}{chapter.5}% 98
\BOOKMARK [2][-]{subsection.5.10.1}{\376\377\0005\000.\0001\0000\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.10}% 99
\BOOKMARK [1][-]{section.5.11}{\376\377\0005\000.\0001\0001\000\040\000R\000\351\000f\000\351\000r\000e\000n\000c\000e\000\040\000d\000u\000\040\000f\000i\000c\000h\000i\000e\000r\000\040\000s\000t\000r\000u\000c\000t\000u\000r\000e\000.\000h\000p\000p}{chapter.5}% 100
\BOOKMARK [2][-]{subsection.5.11.1}{\376\377\0005\000.\0001\0001\000.\0001\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000\351\000t\000a\000i\000l\000l\000\351\000e}{section.5.11}% 101
\BOOKMARK [0][-]{section*.28}{\376\377\000I\000n\000d\000e\000x}{}% 102
